robocorp-code/vscode-client/src/viewsWorkItems.ts

```diff
@@ -0,0 +1,203 @@
+import * as vscode from 'vscode';
+import { resolve, join, dirname, basename } from 'path';
+
+import { logError } from './channel';
+import { ROBOCORP_LIST_WORK_ITEMS_INTERNAL } from "./robocorpCommands";
+import { FSEntry, RobotEntry, treeViewIdToTreeView } from "./viewsCommon";
+import { TREE_VIEW_ROBOCORP_ROBOTS_TREE, TREE_VIEW_ROBOCORP_WORK_ITEMS_TREE } from './robocorpViews';
+import { getCurrRobotDir, RobotSelectionTreeDataProviderBase } from './viewsRobotSelection';
+
+const WORK_ITEM_TEMPLATE = `[
+  {
+      "payload": {
+         "message": "Hello World!"
+      },
+      "files": {
+          "orders.xlsx": "orders.xlsx"
+      }
+  }
+]`;
+
+async function getWorkItemInfo(): Promise<WorkItemsInfo | null> {
+  // Would there be a faster way of getting the work item path other than querying the work item info?
```

Comment:
Well, since we're dealing with the tree and the related command is hidden from the command pallette, you could get it  (with `treeViewIdToTreeView.get(TREE_VIEW_ROBOCORP_WORK_ITEMS_TREE);` and then have some getter for:

`WorkItemsTreeDataProvider.workItemsInfo`.

It does make more sense to just use the info already cached in the tree in this case.

Although note that if we decided to show the command in the command palette later on, the code in this method would be needed as a fallback, so, we may want to keep it in the case where the tree wouldn't be available...